changelog.md

```diff
@@ -1,5 +1,10 @@
 # Chaangelog
 
+## 8.7.0
+
+* userService.getUser
+* userService.getUserByPerson
```

Comment:
Can you mention the new entity tree here?

Reply:
Yeah I think it should get a new minor number but I'll leave the release process to @TehWazzard.

Ah didn't realize the meta key. Yes I can see how just Left/Right could be confusing. Just tried it and I think it's fine with the meta key - it's a power user feature after all!

Reply:
Yeah, I will do the version bump once we merge :)

Reply:
checking if this point is resolved. if yes I can start merging.

Reply:
Changelog hasn't been updated yet but that shouldn't prevent this from being merged - can always be updated later.

However I'm still not clear on this:

> I'm still not sure how we would go about showing the full tree without any filtering - I would expect `<ws-entity-tree-select></ws-entity-tree-select>` but that turns up empty? (added in the tester now)

Reply:
When entities === null (by default) it shows all entities